apps/microtvm/arduino/template_project/src/example_project/project.ino

```diff
@@ -17,10 +17,10 @@
  * under the License.
  */
 
-#include "src/model.h"
+#include "src/standalone_crt/include/tvm/runtime/crt/platform.h"
 
 void setup() {
-  TVMInitialize();
+  TVMPlatformInitialize();
```

Comment:
do we want to elevate TVMPlatform to an API that the user should call? i think TVMInitialize was arduino-specific...on the other hand i understand that TVMInitialize's job was to make the other TVMPlatform APIs work...so maybe this is ok. i think what i'm wondering is about how TVMInitialize moved out of platform.h. cc @alanmacd

Reply:
TVMInitialize was defined only in Arduino and some other standalone cases. My goal was to make this call more generic to implement all platform specific initialization that are necessary for other APIs. Things like memory initialization, timer, etc.

Reply:
I like this approach of being very high level. I know we haven't focused a whole lot on usability, but being able to call one function and set up the whole thing is really nice.